src/components/EmailPasswordForm/EmailPasswordForm.tsx

```diff
@@ -62,7 +80,7 @@ function EmailPasswordForm({
           value={signUpData.password}
         />
       </div>
-      <Button typeRole='submit' label='Sign In' style='primary' />
+      <Button typeRole='submit' label={submitLabel} style='primary' />
```

Comment:
Minor: I think that we can replace the `label` property with `children` (`JSX.Element`). This way, we can use the `Button` in a more natural way:

```typescript
<Button>
   {label}
</Button>
```

Also, we can try to follow the HTML5 properties. For instance, instead of `typeRole` we can use plain `role`.